demos/speech_recognition_wav2vec_demo/python/speech_recognition_wav2vec_demo.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+"""
+ Copyright (C) 2018-2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from argparse import ArgumentParser, SUPPRESS
+from itertools import groupby
+import logging as log
+from time import perf_counter
+import sys
+
+import numpy as np
+import wave
+
+from openvino.inference_engine import IECore, get_version
+
+log.basicConfig(format='[ %(levelname)s ] %(message)s', level=log.DEBUG, stream=sys.stdout)
+
+def build_argparser():
+    parser = ArgumentParser(add_help=False)
+    parser.add_argument('-h', '--help', action='help', default=SUPPRESS, help='Show this help message and exit.')
+    parser.add_argument('-m', '--model', help='Required. Path to an .xml file with a trained model.', required=True)
+    parser.add_argument('-i', '--input', help="Required. Path to an audio file in WAV PCM 16 kHz mono format", required=True)
+    parser.add_argument('-d', '--device', default='CPU',
+                        help="Optional. Specify the target device to infer on, for example: "
+                             "CPU, GPU, HDDL, MYRIAD or HETERO. "
+                             "The demo will look for a suitable IE plugin for this device. Default value is CPU.")
+    return parser
+
+
+class Wav2Vec:
+    alphabet = [
+        "[pad]", "[s]", "[s]", "[unk]", "|",
+        "e", "t", "a", "o", "n", "i", "h", "s", "r", "d", "l", "u",
+        "m", "w", "c", "f", "g", "y", "p", "b", "v", "k", "'", "x", "j", "q", "z"]
+    words_delimiter = '|'
+    pad_token = '<pad>'
```

Comment:
I'm confused: in alphabet you use `[pad]`, here `<pad>`. Is it right?

Reply:
forgot to rename after changes in description

Reply:
fixed